README.md

```diff
@@ -16,9 +19,13 @@
 
 <p style="clear:both;">
 <h2><a href="https://layer5.io/meshery">Meshery</a></h2>
-<a href="https://meshery.io"><img src="https://raw.githubusercontent.com/meshery/meshery-cpx/master/img/readme/meshery-logo-light-text.svg"
-style="margin:10px;" width="125px" 
-alt="Meshery - the Service Mesh Management Plane" align="left" /></a>
+<a href="https://meshery.io">
+<picture><img src="./img/readme/meshery-logo-light-text.svg#gh-dark-mode-only"
+style="margin:10px;" width="10%" 
+alt="Meshery - the Service Mesh Management Plane" align="left" />
```

Comment:
```suggestion
alt="Meshery - the Cloud Native Manager" align="left" />
```

Reply:
Meshery started as the service mesh manager, but has vastly expanded past service mesh to all of cloud native.